.editorconfig

```diff
@@ -23,4 +33,10 @@ insert_final_newline=true
 indent_style=space
 indent_size=2
```

Comment:
As you mentioned we're keeping `indent_size=4`, but here it seems 2.

Reply:
This is for shell files.